specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManager.json

```diff
@@ -0,0 +1,682 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Network Manager": {
+            "$ref": "./examples/NetworkManagerList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/effectiveVirtualNetworks": {
+      "post": {
+        "tags": [
+          "NetworkManagers"
+        ],
+        "operationId": "EffectiveVirtualNetworks_ListByNetworkManager",
+        "description": "List effective virtual networks in a network manager.",
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ListPageSizeParameter"
+          },
+          {
+            "$ref": "#/parameters/ListSkipTokenParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/EffectiveVirtualNetworksParameter"
+            },
+            "description": "Effective Virtual Networks Parameter."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns a list of effective Virtual Networks.",
+            "schema": {
+              "$ref": "./networkManagerGroup.json#/definitions/EffectiveVirtualNetworksListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Network Manager": {
+            "$ref": "./examples/NetworkManagerEffectiveVirtualNetworksListByNetworkManager.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkManager": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkManagerProperties",
+          "description": "The network manager properties"
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system metadata related to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
```

Comment:
v2 spec is latest. Please refer this https://github.com/Azure/azure-rest-api-specs/blob/92aa7802beffd6bdae18b808e45c4897de2cb2e2/specification/common-types/resource-management/v2/types.json#L450

Reply:
Please check that all proxy resources have top level property SystemData

---
In reply to: [574193358](https://github.com/Azure/azure-rest-api-specs/pull/12441#discussion_r574193358) [](ancestors = 574193358)